### DISCUSSION TITLE
Setup over terminal without web browser

### DISCUSSION THREAD

**Discussion by Cilenco** [AdguardTeam/AdGuardHome#4512] [Q&A]
Hey together,

is there a way to setup the service over a terminal without the requirement of a web browser?
In a quick search I couldn't find anything in this direction. Any help would be great :)

Thanks in advance!

**Comment by ainar-g** [comment]
Hello.  #4199 is basically about that.  Being able to provide the data necessary for the initial setup from the command line.
